### PR TITLE
General: Use query functions in integrator

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Brief description
New integrator plugin is using query functions from `openpype.client`.

## Description
Integrator PR was created before query functions were available (legacy integrator is already using them).

## Testing notes:
1. New integrator should work as did before but is using query functions

### CODE REVIEW

File: openpype/plugins/publish/integrate.py [hunk 1]
```diff
@@ -289,19 +296,17 @@ def register(self, instance, file_transactions, filtered_repres):
 
         template_name = self.get_template_name(instance)
 
-        subset, subset_writes = self.prepare_subset(instance)
+        subset, subset_writes = self.prepare_subset(instance, project_name)
         version, version_writes = self.prepare_version(instance, subset)
```
Comment (antirotor) [inline openpype/plugins/publish/integrate.py:300]: Is this missing argument?

```suggestion
        version, version_writes = self.prepare_version(instance, subset, projet_name)
```

**Review by antirotor — CHANGES_REQUESTED** [review]
It is failing with this traceback:

```
Traceback (most recent call last):
  File "C:\Users\annat\Documents\pype\3.0\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "C:\Users\annat\Documents\pype\3.0\openpype\plugins\publish\integrate.py", line 214, in process
  File "C:\Users\annat\Documents\pype\3.0\.venv\lib\site-packages\six.py", line 719, in reraise
    raise value
  File "C:\Users\annat\Documents\pype\3.0\openpype\plugins\publish\integrate.py", line 208, in process
  File "C:\Users\annat\Documents\pype\3.0\openpype\plugins\publish\integrate.py", line 300, in register
TypeError: prepare_version() missing 1 required positional argument: 'project_name'
```

File: openpype/plugins/publish/integrate.py (hunk 1 above)
Comment (iLLiCiTiT) [reply openpype/plugins/publish/integrate.py]: Yep, didn't commit that.

**Review by kalisp — APPROVED** [review]
Tested in AE, PS, Nuke

**Review by antirotor — APPROVED** [review]
It works in Maya too, hooray!

**Pull request merged**